### PR TITLE
Prepare packaging for v4.0

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

**Comment by EvgeniyPatlan** [comment]
@tplavcic I have updated license in rpm spec file.

File: percona-packaging/debian/changelog [hunk 1]
```diff
@@ -1,5 +1,5 @@
-percona-server-mongodb-36 (3.6.0) UNRELEASED; urgency=medium
+percona-server-mongodb (4.0.3) UNRELEASED; urgency=medium
```
Comment (igorsol) [inline percona-packaging/debian/changelog]: The first GA release will be at least 4.0.4, not 4.0.3.

File: percona-packaging/debian/changelog [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 
   * Updated to next upstream release. (Closes: #RM-564)
 
- -- Evgeniy Patlan <evgeniy.Patlan@percona.com>  Fri, 15 Dec 2017 12:20:22 +0600
+ -- Evgeniy Patlan <evgeniy.Patlan@percona.com>  Wed, 21 Dec 2017 12:20:22 +0600
```
Comment (igorsol) [inline percona-packaging/debian/changelog]: Still `Dec 2017` ?

File: percona-packaging/redhat/percona-server-mongodb.spec.template [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 #
-Name:           Percona-Server-MongoDB-36
+Name:           Percona-Server-MongoDB
 Version:        @@VERSION@@
 Release:        @@RELEASE@@%{?dist}
 Summary:        High-performance MongoDB replacement from Percona (metapackage)
 Group:          Applications/Databases
 
-License:        AGPL
+License:        AGPL 3.0
```
Comment (igorsol) [inline percona-packaging/redhat/percona-server-mongodb.spec.template]: upstream uses SSPL. Are you sure we still on AGPL?

File: percona-packaging/debian/changelog (hunk 1 above)
Comment (EvgeniyPatlan) [reply percona-packaging/debian/changelog]: It is ok as current version is 4.0.3.

File: percona-packaging/redhat/percona-server-mongodb.spec.template (hunk 3 above)
Comment (EvgeniyPatlan) [reply percona-packaging/redhat/percona-server-mongodb.spec.template]: According to https://github.com/mongodb/mongo/blob/v4.0/rpm/mongodb-org-unstable-init.spec#L7
it is still AGPL

File: percona-packaging/debian/changelog (hunk 2 above)
Comment (EvgeniyPatlan) [reply percona-packaging/debian/changelog]: fixed

File: percona-packaging/debian/changelog (hunk 1 above)
Comment (EvgeniyPatlan) [reply percona-packaging/debian/changelog]: fixed

**Pull request merged**